dll/cpl/sysdm/resource.h

```diff
@@ -63,6 +63,9 @@
 
 #define IDS_UPTIME_FORMAT 140
 
+#define IDS_EVIRONMENT_WARNING            150
+#define IDS_EVIRONMENT_WARNING_TITLE      151
```

Comment:
```suggestion
#define IDS_ENVIRONMENT_WARNING            150
#define IDS_ENVIRONMENT_WARNING_TITLE      151
```
(to be fixed everywhere else)